recipe/meta.yaml

```diff
@@ -54,10 +54,10 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib('c') }}
```

Comment:
We tried this in https://github.com/conda-forge/lammps-feedstock/pull/198 but that was not successful so far, so for the moment we just ignore the recommendation.

Reply:
ok, let me try removing it again